lib/membrane_live/event.ex

```diff
@@ -230,6 +230,9 @@ defmodule MembraneLive.Event do
 
       pid == state.moderator_pid ->
         state = %{state | moderator_pid: nil}
+        {:ok, state} = handle_peer_left(state, pid)
+
+        Engine.remove_peer(state.rtc_engine, pid)
```

Comment:
Hmmm I am not sure about this because, `Engine.remove_peer` sends `Message.PeerLeft`, and in the handler for this message, we call `handle_peer_left again`

Reply:
Right, I will look into that.